en/components/carousel.md

```diff
@@ -341,7 +341,41 @@ These configurions will have the following result:
 </div>
 
 
-
+## Accessibility
+### WAI-ARIA Roles, States, and Properties
+ * The Carousel base element role is [`region`](https://www.w3.org/TR/wai-aria-1.1/#region) - section containing content that is relevant to specific purpose and users will likely want to be able to navigate easily.
+ * Carousel indicators are with role [`tab`](https://www.w3.org/TR/wai-aria-1.1/#tab) -  grouping label providing a mechanism for selecting the tab content that is to be rendered to the user
+ * The element that serves as the container for the set of tabs (carousel indicators) role is set to [`tablist`](https://www.w3.org/TR/wai-aria-1.1/#tab).
+ * Each slide element is set with role [`tabpanel`](https://www.w3.org/TR/wai-aria-1.1/#tabpanel).
+ * The element that serves as the container for the set of igx-slides is set with [aria-live](https://www.w3.org/TR/wai-aria-1.1/#aria-live)="polite". Both options are
+   - `off`: if the carousel is automatically rotating.
+   - `polite`: if the carousel is NOT automatically rotating.
+
+### ARIA support
+#### **Carousel component**
+##### `Roles`: 
+ * [role="region"](https://www.w3.org/TR/wai-aria-1.1/#region)
+ * [role="button"](https://www.w3.org/TR/wai-aria/roles#button)
+ * [role="tablist"](https://www.w3.org/TR/wai-aria-1.1/#tablist)
+ * [role="tab"](https://www.w3.org/TR/wai-aria-1.1/#tab)
+##### `Attributes`:   
+ * [aria-roledescription](https://www.w3.org/TR/wai-aria-1.1/#aria-roledescription)='carousel'
+ * [aria-selected](https://www.w3.org/TR/wai-aria/states_and_properties#aria-selected)="slide.active" 
+ * [aria-controls](https://www.w3.org/TR/wai-aria-1.1/#aria-controls)="'panel-' + slide.index" 
+ * [aria-live](https://www.w3.org/TR/wai-aria-1.1/#aria-live)="!interval || stoppedByInteraction ? 'polite' : 'off'" 
+ * [aria-label](https://www.w3.org/TR/wai-aria/states_and_properties#aria-label)="carousel" 
+ * aria-label="setAriaLabel(slide)" - slide based
+ * aria-label (buttons)
+   - aria-label="resourceStrings.igx_carousel_previous_slide" (keydown.enter)="prev()"
+   - aria-label="resourceStrings.igx_carousel_next_slide" (keydown.enter)="next()"
```

Comment:
maybe here is better instead of the snippets from the actual implementation to explain just why and what is the actual value to which these properties are set:
e.g [aria-selected] - set to true or false based on the active slide. Provides additional information about a selection interaction like the current tab